coral/src/domain/auth-user/auth-user-api.ts

```diff
@@ -25,7 +49,7 @@ function transformAuthResponse(response: KlawApiResponse<"getAuth">): AuthUser {
   };
 }
 
-function getAuth(): Promise<AuthUser> {
+async function getAuth(): Promise<AuthUser> {
```

Comment:
I would use the `async` keyword only if there is `await` used in the function body.

Reply:
Ah, thanks for catching! I awaited the user before here for testing, forgot to remove the asyn 🙈

Reply:
Updated! https://github.com/Aiven-Open/klaw/pull/1587/commits/1e708741733f82fa96385ac78558ff693a526771